src/core/exec-invoke.c

```diff
@@ -72,6 +76,11 @@
 
 #define SNDBUF_SIZE (8*1024*1024)
 
+/* Project id range for disk quotas */
+#define PROJ_ID_MIN 2147483648
+#define PROJ_ID_MAX 4294967294
+#define PRJID_CLAMP_INTO_QUOTA_RANGE(rnd) (((uint32_t) (rnd) % (PROJ_ID_MAX - PROJ_ID_MIN + 1)) + PROJ_ID_MIN)
```

Comment:
please stick to one way to spell the thing, i.e. either PROJ_ID or PRJID or PROJECT_ID, but let's not name this differently in every other identifier like you are doing here.

Reply:
i think it would make sense to cast to uint32_t *after* the modulo operation, not before, since siphash24() returns a 64bit value, and if you write it like the above you hence basically calculate `rnd % (2^32) % (PROJ_ID_MAX-PROJID_MIN+1))` and that's a bit weird.